leits/MeetingBar
# Support detection of zoom.com meeting links

### Status
**READY*

### Description
This change adds support for meetings using subdomains of zoom.com.


## Checklist

### Steps to Test or Reproduce
1. Create a meeting with the following meeting link: https://testcompany.zoom.com/j/92313847288?pwd=cY792Kdy4js92JA90kasYhTsL26Jhjwx8sg
2. Select the event form the list of meetings
3. The meeting link should open (note this is not a real meeting link).
